### PR TITLE
Updated the descriptions on all csproj files

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
Now all files follow the same standard which will reflect in the corresponding nuget packages

```xml
    <Description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework. The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.

This package provides you with MonoGame Framework that works on Android.</Description>
```

Resolves: #7259

### CODE REVIEW

**Comment by tomspilman** [comment]
@SimonDarksideJ @Jjagg @harry-cpp -  I would like to see us reverse this description or maybe change what we put in it.

Currently you see this....

image

The long marketing message at the top of the description becomes noise and you cannot see the actual useful information... what the package is for.   Even in VS you get noise....

image

However note that last one is very clear.

I recommend we simply remove the "marketing crap" from the top of the descriptions.  Instead the description tells the user what this package is for,  We should assume they know what MonoGame is or are interested in finding out.

**Comment by SimonDarksideJ** [comment]
I'm fine with that. So long as we pick a standard and stick to it.  Update in coming

**Comment by tomspilman** [comment]
@SimonDarksideJ - We can also avoid the redundant `This package provides you` bit.

**Comment by SimonDarksideJ** [comment]
How does this sound @tomspilman (before I edit all the files again)

```xml
    <Description>MonoGame Framework built for Android.
    
    For more details about MonoGame please visit https://MonoGame.Net</Description>
```

**Comment by tomspilman** [comment]
> How does this sound @tomspilman (before I edit all the files again)

I think the main issue with that is that the preview will ignore your newline and squish the two lines back together. 

Second.... `MonoGame Framework built for Android.` could maybe include more useful info?  Like...

 > MonoGame runtime framework for all Android 4.2 or newer devices.

 (i think that statement is correct)

Third the package already contains our url to the site.  This extra link is redundant to me and isn't what people are looking for when searching for our packages.

More a nitpick on my end... but IMO `https://MonoGame.Net` is super ugly.  We should just have `www.monogame.net` and not camel case the url which is a case insensitive thing.

**Comment by SimonDarksideJ** [comment]
After reviewing some other Microsoft packages for how they are working the descriptions for their NuGet packages

```
This package contains the core runtime assemblies for the MonoGame framework to support Android 4.2 or newer devices.
```

Dropped the url's and other info, however the nuspecs must be built of a template on the build server, so if you want to update the URL's in the NuGets themselves, it will have to be updated there.  Doesn't exist in the repository.

**Comment by tomspilman** [comment]
@SimonDarksideJ 

Sure i can live with that... ton better that where we started.

File: MonoGame.Framework/MonoGame.Framework.iOS.csproj [hunk 1]
```diff
@@ -5,13 +5,11 @@
     <DefineConstants>IOS;GLES;STBSHARP_INTERNAL</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <MGOpenGL>GLES</MGOpenGL>
-
-    <Description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework. The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.
-
-This package provides you with MonoGame Framework that works on iOS.</Description>
+    <Description>This package contains the core runtime assemblies for the MonoGame framework to support iOS Versions 10 and above </Description>
```
Comment (Jjagg) [inline MonoGame.Framework/MonoGame.Framework.iOS.csproj:8]:
```suggestion
    <Description>The MonoGame runtime for iOS. Supports iOS 10 and newer.</Description>
```

File: Templates/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj [hunk 2]
```diff
@@ -2,18 +2,15 @@
   <PropertyGroup>
     <BaseOutputPath>..\..\Artifacts\MonoGame.Templates.CSharp</BaseOutputPath>
     <PackageType>Template</PackageType>
-
     <Title>MonoGame project templates</Title>
-    <Description>A set of C# templates written for the MonoGame framework.</Description>
+    <Description>This package contains a set of C# templates written for the MonoGame framework.</Description>
```
Comment (Jjagg) [inline Templates/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj:6]: I prefer how this was before.

File: Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Linux.csproj [hunk 3]
```diff
@@ -3,10 +3,12 @@
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package provides you with a graphical editor for the MonoGame Framework Content Builder, used for editing your content projects ready for processing.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Linux.csproj:6]:
```suggestion
    <Description>A graphical editor for the MonoGame Content Builder (MGCB). Used for editing your content projects.</Description>
```

File: Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Mac.csproj [hunk 4]
```diff
@@ -3,12 +3,14 @@
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+     <Description>This package provides you with a graphical editor for the MonoGame Framework Content Builder, used for editing your content projects ready for processing.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Mac.csproj:6]:
```suggestion
     <Description>A graphical editor for the MonoGame Content Builder (MGCB). Used for editing your content projects.</Description>
```

File: Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Windows.csproj [hunk 5]
```diff
@@ -4,11 +4,13 @@
     <OutputType>WinExe</OutputType>
     <UseWPF>true</UseWPF>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package provides you with a graphical editor for the MonoGame Framework Content Builder, used for editing your content projects ready for processing.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.Windows.csproj:7]:
```suggestion
    <Description>A graphical editor for the MonoGame Content Builder (MGCB). Used for editing your content projects.</Description>
```

File: Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.csproj [hunk 6]
```diff
@@ -3,10 +3,12 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package provides you with a graphical editor for the MonoGame Framework Content Builder, used for editing your content projects ready for processing.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Content.Builder.Editor/MonoGame.Content.Builder.Editor.csproj:6]:
```suggestion
    <Description>A graphical editor for the MonoGame Content Builder (MGCB). Used for editing your content projects.</Description>
```

File: Tools/MonoGame.Content.Builder.Task/MonoGame.Content.Builder.Task.csproj [hunk 7]
```diff
@@ -5,6 +5,8 @@
     <BaseOutputPath>..\..\Artifacts\MonoGame.Content.Builder.Task</BaseOutputPath>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <Description>This package provides additionjal functionality for the content pipeline to process content.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Content.Builder.Task/MonoGame.Content.Builder.Task.csproj:8]:
```suggestion
    <Description>MSBuild task to automatically build content for MonoGame.</Description>
```

File: Tools/MonoGame.Content.Builder/MonoGame.Content.Builder.csproj [hunk 8]
```diff
@@ -3,13 +3,15 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package provides you with the MonoGame Framework Content Builder (MGCB) tool used for packaging content in to xnb files.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Content.Builder/MonoGame.Content.Builder.csproj:6]:
```suggestion
    <Description>The MonoGame Content Builder (MGCB) tool used for packaging content into xnb files.</Description>
```

File: Tools/MonoGame.Effect.Compiler/MonoGame.Effect.Compiler.csproj [hunk 9]
```diff
@@ -3,13 +3,15 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package provides you with MonoGame Framework Effect Compiler (MGFX) tool used for packaging content in to xnb files.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Effect.Compiler/MonoGame.Effect.Compiler.csproj:6]:
```suggestion
    <Description>The MonoGame Framework Effect Compiler (MGFX) tool used to compile shaders for use with MonoGame.</Description>
```

File: Tools/MonoGame.Tools.Tests/MonoGame.Tools.Tests.csproj [hunk 10]
```diff
@@ -3,13 +3,15 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package contains all the Unit Tests for the MonoGame Framework.</Description>
```
Comment (Jjagg) [inline Tools/MonoGame.Tools.Tests/MonoGame.Tools.Tests.csproj:6]:
```suggestion
    <Description>The unit tests for the MonoGame framework.</Description>
```

Don't really need a description for this one, but should be consistent if we add it.

File: MonoGame.Framework/MonoGame.Framework.DesktopGL.csproj [hunk 11]
```diff
@@ -4,11 +4,10 @@
     <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
     <DefineConstants>OPENGL;OPENAL;XNADESIGNPROVIDED;LINUX;DESKTOPGL;SUPPORTS_EFX;NETSTANDARD;STBSHARP_INTERNAL</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework. The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.
-
-This package provides you with MonoGame Framework that uses OpenGL for rendering and works on Windows, Linux and MacOS.</Description>
+    <Description>This package contains the core runtime assemblies for the MonoGame framework that uses OpenGL for rendering and targets Windows, Linux and MacOS.</Description>
```
[the remark this answers is not in the document]
Comment (tomspilman) [reply MonoGame.Framework/MonoGame.Framework.DesktopGL.csproj:7]: Only thing we could add would be something making it clear it produces a single binary that works on all three platforms.  Maybe as simple as adding the word "multiplatform" or "in a single assembly" or something.

File: MonoGame.Framework/MonoGame.Framework.WindowsDX.csproj [hunk 12]
```diff
@@ -5,11 +5,10 @@
     <DefineConstants>WINDOWS;XNADESIGNPROVIDED;STBSHARP_INTERNAL</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <UseWindowsForms>true</UseWindowsForms>
-    <Description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework. The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.
-
-This package provides you with MonoGame Framework that uses DirectX for rendering and works on Windows.</Description>
+    <Description>This package contains the core runtime assemblies for the MonoGame framework that uses DirectX for rendering and targets Windows</Description>
```
[the remark this answers is not in the document]
Comment (tomspilman) [reply MonoGame.Framework/MonoGame.Framework.WindowsDX.csproj:8]: I would combine the sentence on this one.

File: Templates/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj (hunk 2 above)
Comment (tomspilman) [reply Templates/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj:6]: Do we need to specify it is for Visual Studio?  Or are these templates usable for other IDEs like VSCode or Ryder ?

**Comment by SimonDarksideJ** [comment]
Ok @Jjagg / @tomspilman I'll make these final changes, but we do have to draw a line under this :D

File: Tools/MonoGame.Content.Builder/MonoGame.Content.Builder.csproj (hunk 8 above)
Comment (tomspilman) [reply Tools/MonoGame.Content.Builder/MonoGame.Content.Builder.csproj:6]: I would suggest:

`The MonoGame Content Builder (MGCB) command line tool used for optimizing content for runtime use.`

File: Tools/MonoGame.Effect.Compiler/MonoGame.Effect.Compiler.csproj (hunk 9 above)
Comment (tomspilman) [reply Tools/MonoGame.Effect.Compiler/MonoGame.Effect.Compiler.csproj:6]: `The MonoGame Framework Effect Compiler (MGFXC) command line tool used to compile shaders.`

**Comment by tomspilman** [comment]
@SimonDarksideJ maybe give @Jjagg a moment to review my suggestions to his suggestions.

File: Templates/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj (hunk 2 above)
Comment (Jjagg) [reply Templates/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj:6]: They're not tied to any IDE. You run them from the command line. That's true for all template packages on nuget so I don't think we need to specify that.

**Comment by Jjagg** [comment]
Agreed on all suggestions :)

**Comment by SimonDarksideJ** [comment]
Will get these incorporated now.

File: Tools/MonoGame.Tools.Tests/MonoGame.Tools.Tests.csproj (hunk 10 above)
Comment (SimonDarksideJ) [reply Tools/MonoGame.Tools.Tests/MonoGame.Tools.Tests.csproj:6]: It is more for consistency

**Comment by SimonDarksideJ** [comment]
Incorporated latest feedback and applied the same consistency to all the .csproj descriptions based off @Jjagg s updates.

File: MonoGame.Framework/MonoGame.Framework.iOS.csproj [hunk 13]
```diff
@@ -5,13 +5,11 @@
     <DefineConstants>IOS;GLES;STBSHARP_INTERNAL</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <MGOpenGL>GLES</MGOpenGL>
-
-    <Description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework. The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.
-
-This package provides you with MonoGame Framework that works on iOS.</Description>
+    <Description>The MonoGame runtime for iOS which supports iOS Versions 10 and above </Description>
```
Comment (Jjagg) [inline MonoGame.Framework/MonoGame.Framework.iOS.csproj:8]: Missing period at the end here.

Comment (SimonDarksideJ) [reply MonoGame.Framework/MonoGame.Framework.iOS.csproj:8]: Good catch.

**Comment by Jjagg** [comment]
Thanks, Simon!

**Comment by SimonDarksideJ** [comment]
Woot, and thank you @Jjagg for the support!
Good to get all these "branding" tasks out of the way. Next docs :D

**Pull request merged**